cmd/start.go

```diff
@@ -58,6 +58,8 @@ func init() {
 	startCmd.Flags().String("helm-ns", "default", "Namespace for resources deployed by helm")
 	startCmd.Flags().String("helm-prefix", "lostromos", "Prefix for release names in helm")
 	startCmd.Flags().String("helm-tiller", "tiller-deploy:44134", "Address for helm tiller")
+	startCmd.Flags().Bool("helm-wait", false, "Use the helm --wait flag for creating and updating releases")
+	startCmd.Flags().Int64("helm-wait-timeout", 120, "The time in seconds to wait for kubernetes resources to be created when doing a helm install or upgrade")
```

Comment:
This should be an int not an int64

Reply:
Its an int64 because the helm.InstallTimeout option requires an int64.

Reply:
:( okay